egs/wsj/s5/steps/select_feats.sh

```diff
@@ -22,7 +22,7 @@ echo "$0 $@"  # Print the command line for logging
 if [ -f path.sh ]; then . ./path.sh; fi
 . parse_options.sh || exit 1;
 
-if [ $# -ne 5 ]; then
+if [ $# -lt 3 ] || [ $# -gt 5 ]; then
    echo "usage: $0 [options] <selector> <src-data-dir>  <dest-data-dir> <log-dir> <path-to-storage-dir>";
```

Comment:
You have changed the usage but you have not adjusted the usage message appropriately.